contrib/kind-common

```diff
@@ -0,0 +1,369 @@
+if [ "${BASH_SOURCE[0]}" -ef "$0" ]
+then
+    >&2 echo 'Please source this script, not execute it!'
+    exit 1
+fi
+
+ARCH=""
+case $(uname -m) in
+    x86_64)  ARCH="amd64" ;;
+    aarch64) ARCH="arm64"   ;;
+esac
+
+function if_error_exit {
```

Comment:
Note: inconsistent function format, but decided to leave it unchanged to make review easier to check